resources/postcards/models/postcard_editable.yml

```diff
@@ -12,13 +12,46 @@ allOf:
 
     properties:
       front:
+        description: >
```

Comment:
This works, but it's weird that it _does_, given [this bug](https://github.com/TOMP-WG/TOMP-API/issues/275). I'm thinking that the `oneOf` changes things. Either way, 🤷🏽